build-system/tasks/issues-tasks.js

```diff
@@ -0,0 +1,255 @@
+  },
+};
+
+if (GITHUB_ACCESS_TOKEN) {
+  issuesopt.qs = {
+    access_token: GITHUB_ACCESS_TOKEN
+  }
+}
+
+if (GITHUB_ACCESS_TOKEN) {
+  optionsMilestone.qs = {
+    access_token: GITHUB_ACCESS_TOKEN
+  }
+}
+
+function processissues() {
+  if (!GITHUB_ACCESS_TOKEN) {
+    util.log(util.colors.red('You have not set the ' +
+        'GITHUB_ACCESS_TOKEN env var.'));
+    util.log(util.colors.green('See https://help.github.com/articles/' +
+        'creating-an-access-token-for-command-line-use/ ' +
+        'for instructions on how to create a github access token. We only ' +
+        'need `public_repo` scope.'));
+    return;
+  }
+
+  return getGitMetadata();
+}
+
+function getGitMetadata() {
+return getIssues()
+.then(function() {
+	util.log(util.colors.blue('automation applied'));
+	if (isDryrun) {
+    return;
+  }
+})
+.catch(errHandler);	
+}
+/**
+ * Function goes through all the gitHub issues,
+ * gets all the Labels we are interested in,
+ * depending if missing milestone or label, 
+ * tasks applied as per design go/ampgithubautomation
+ */
+function getIssues(){
+  var promise = Promise.resolve();
+  var options = extend({}, issuesopt);
+  options.qs = {
+    state: 'open',
+    per_page: 1000,
+    access_token: GITHUB_ACCESS_TOKEN,
+  };
+  return request(options).then(res => {
+    const issues = JSON.parse(res.body);
+    issues.forEach(function(issue) {
+      var labels = issue.labels;
+      var issueType;
+      var milestone = issue.milestone;
+      var milestoneTitle; 
+      var milestoneState;
+      var hasPriority = false;
+
+      // Get the title and state of the milestone 
+      if (milestone) {
+        milestoneTitle = milestone['title']; 
+        milestoneState = milestone['state'];
+      }
+      // Get the labels we want to check
+      labels.forEach(function(label) {
+        if (label) {
+          // Check if the issues has type
+          if (label.name.startsWith('Type') || label.name.startsWith('Related'))
+            issueType = label.name;
+          // Check if the issues has Priority
+          if (label.name.startsWith('P0') || label.name.startsWith('P1') || 
+            label.name.startsWith('P2') || label.name.startsWith('P3'))
+            hasPriority = true;
+        }
+      });
+      promise = promise.then(function() {
+        util.log('Update ' + issue.number);
+        var updates = [];
+        // Milestone task: move issue from closed milestone
+        if (milestone) {
+          if (milestoneTitle.startsWith('Sprint') && milestoneState == 'closed') {
+            // 4 is the number for Milestone 'Backlog Bugs' 
+            updates.push(applyMilestone(issue, 4));
+          }
+        }
+        //if issueType is not null, add correct milestones
+        if (issueType != null) {
+          if (milestoneTitle == 'Pending Triage' || milestone == null) {
+            if (issueType == 'Type: Feature Request') {
+              // 23 is the number for Milestone 'New FRs' 
```

Comment:
It's best to move this constants on top and document there. Then use it as:

```
applyMilestone(issue, ISSUE_NEW_FR);
```

Reply:
done